### ISSUE TITLE
ShimmeredDetailsList shows vertical shimmer bars when resized to 90% zoom

### ISSUE THREAD

**Issue by dehoward** [microsoft/fluentui#12396]
### Environment Information

- **Package version(s)**: 7.105.1
- **Browser and OS versions**: MS Edge Version 80.0.361.66, Windows 10 Version 1909

### Please provide a reproduction of the bug in a codepen:
When resizing the ShimmeredDetailsList from 100% to 90% zoom, vertical shimmer bars are shown next to each column. This can be observed in the [DetailsList Shimmer Example](https://docs.microsoft.com/en-us/javascript/api/examples/detailslist/shimmer?view=office-ui-fabric-react-latest#detailslist-shimmer-example).

#### Actual behavior:
Red rectangle added for emphasis 
![bug](https://user-images.githubusercontent.com/52973358/77371876-8b802180-6d21-11ea-9b79-f08827e6f688.PNG)

#### Expected behavior:

No vertical bars are shown.

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)
No

Requested priority: (Blocking, High, Normal, Low)
Normal 

Products/sites affected: (if applicable)

**Comment by Vitalius1** [comment]
@msdehoward I looked succinctly into the issue because I am owning this control and apparently this is different depending in what browser you are. In Chrome and Edge Chromium I can see the vertical bar you describe only on the last column and in your screenshot it appears to be on every column:
![image](https://user-images.githubusercontent.com/29514833/77383709-da897f00-6d40-11ea-878d-a643bb9a60e6.png)

Also, what I noticed is that if you go to 80% zoom and then back to 90% the bar disappears. It only comes back when going down from 100% to 90%. This makes me think this is some browser quirks with how it does the scaling math during zoom. Not sure if this is something we can fix. Will try and give it a quick look but as of now this will be a low priority issue. Let me know if the verticals disappear if you go to 80% and then back to 90%.

**Comment by Vitalius1** [comment]
I took a look and I can validate that this is a browser quirk because if you get the visual glitches after zooming in or out and then refresh the page you will see that the glitches disappear. Not sure if there is anything we can do to fix this. I couldn't get a consistent repro and even adjusting some numbers internally all I was able to achieve is to move the glitch to a different column. If I opened dev tools on the side of the page while having the glitch it suddenly disappears.

So many indicators that this is a browser issue more than anything and due to this not reproing under normal 100% and not even under many other zoom levels I would say we can close this issue.
